### ISSUE TITLE
 Keyboard problems with some games

### ISSUE THREAD

**Issue by aminta** [MiSTer-devel/Apple-II_MiSTer#12]
I'm trying to play with various Transylvania versions on Apple II Core but I've got a problem: when I try to type in the commands the keyboard is like "stuck" and the letter appears only after many seconds. The (bluetooth) keyboard works perfectly well in all cores (including Apple II core) and even in Transylvania game when, at the start of the game, you're asked to insert your name. The same game files work perfectly well on emulators.

Below the attached video :https://www.icloud.com/sharedalbum/it-it/#B0vG0ehgLGGis0e

Maybe this is related: https://news.ycombinator.com/item?id=20404825

**Comment by steven-a-wilson** [comment]
Something similar was reported with Karateka and Nox Archaist in the ProDOS HDD issue (#5). I briefly looked into it at the time but didn't find any obvious problem. Basically, it seems like any given keystroke _might_ be recognized; it doesn't appear to depend on variables like how many keystrokes there were or how long the key was held (apart from the fact that holding the key long enough engages the auto-repeat function). I played with SignalTap a bit, but I'm not sure I've found any relevant information yet.
